Automattic/jetpack
# Always display status indicator on product cards on My Jetpack

## Proposed changes:

* To improve consistency, this PR updates the product cards on My Jetpack to always display the status indicator (Active, Inactive, Error, etc). 
<img width="1091" alt="image" src="https://github.com/Automattic/jetpack/assets/5714212/0c4eb500-75fc-4c61-ad39-31cb31ced05d">


### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* Spin up a JN site with Jetpack Beta pointing to this branch
* Visit My Jetpack and confirm that all product cards now have the status indicator
* Click to Learn More, activate, and other things and make sure everything works as expected

Note: It's worth checking CRM, which is possibly the only one that never displayed the status indicator (unless the user had CRM installed and active)
